<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchData</class>
 <widget class="QDialog" name="SearchData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>541</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>输入匹配数据</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>521</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>成绩：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="led_SearchPost"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>职位：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>学号：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="led_SearchNum"/>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>姓名：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="led_SearchScore"/>
     </item>
     <item row="0" column="4">
      <widget class="QLineEdit" name="led_SearchName"/>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>其他：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLineEdit" name="led_SearchOther"/>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>排名：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLineEdit" name="led_SearchRange"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>521</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>取消</string>
       </property>
       <property name="icon">
        <iconset resource="iamges.qrc">
         <normaloff>:/images/706.bmp</normaloff>:/images/706.bmp</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbt_Search">
       <property name="text">
        <string>匹配数据</string>
       </property>
       <property name="icon">
        <iconset resource="iamges.qrc">
         <normaloff>:/images/chazhao.ico</normaloff>:/images/chazhao.ico</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="iamges.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SearchData</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbt_Search</sender>
   <signal>clicked()</signal>
   <receiver>SearchData</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
